packs/opentelemetry_collector/variables.hcl

```diff
@@ -56,6 +56,12 @@ variable "privileged_mode" {
   default     = false
 }
 
+variable "use_volumes" {
+  description = "Determines if OTel Collector config is mounted via perstent storage"
```

Comment:
Hum...not sure if this description is quite accurate given how the value is being used.

When `use_volumes` is `true`, the Otel Collector config file is mounted from the alloc into the Docker container, so it's not really a persistent storage. 

Does the Otel Collector need some kind of stateful storage?

Reply:
No, it doesn't need stateful storage. But the original pack had it mounted to stateful storage, which I've never used myself. Wanted to give users the option to do this (i.e. go with the way defined in the original pack, or go with the way that was familiar to me).

Reply:
Ops, sorry I didn't realize that I commented about the same thing twice 😅 

The [other thread](https://github.com/hashicorp/nomad-pack-community-registry/pull/128#discussion_r881062495) has some more details, so I'm going to resolve this one.